go/vt/sqlparser/ast_test.go

```diff
@@ -519,15 +519,6 @@ func TestReplaceExpr(t *testing.T) {
 	}, {
 		in:  "select * from t where convert((select a from b) using utf8)",
 		out: "convert(:a using utf8)",
-	}, {
-		in:  "select * from t where match((select a from b), 1) against (a)",
-		out: "match(:a, 1) against (a)",
-	}, {
-		in:  "select * from t where match(1, (select a from b), 1) against (a)",
-		out: "match(1, :a, 1) against (a)",
-	}, {
-		in:  "select * from t where match(1, a, 1) against ((select a from b))",
-		out: "match(1, a, 1) against (:a)",
```

Comment:
MySQL parses non of these queries so it doesn't really make sense we test these here.